app.rb

```diff
@@ -0,0 +1,228 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'sinatra/content_for'
+require 'sinatra/multi_route'
+require 'staccato'
+require 'sprockets'
+require 'sprockets-helpers'
+require 'json'
+
+require './models/deskpro'
+
+# App is the main Sinatra application
+class App < Sinatra::Base
+
+	set :sprockets, Sprockets::Environment.new
+	set :title, 'Gov.UK Platform as a Service'
+	enable :sessions
```

Comment:
Sessions are no longer required ... hangover from the previous session-based forms